log-insights/setup/aiven/03_configure_collector.mdx

```diff
@@ -0,0 +1,55 @@
+---
+title: 'Step 3: Configure the Collector'
+install_track_title: 'Log Insights: Setup (Aiven)'
+backlink_href: /docs/log-insights/setup
+backlink_title: 'Log Insights: Setup'
+---
+
+import imgGcpFirewallRules from './gcp-firewall-rules.png'
+import imgAwsSecurityGroup from './aws-security-group.png'
+
+export const ImgGcpFirewallRules = () => <img alt="GCP firewall rules allowing access on syslog port" src={imgGcpFirewallRules} />
+
+export const ImgAwsSecurityGroup = () => <img alt="AWS Security Group allowing access on syslog port" src={imgAwsSecurityGroup} />
+
+We can now configure the syslog receiver in the collector. Edit the collector
+config and add the
+[`db_log_syslog_server`](https://pganalyze.com/docs/collector/settings#self-managed-servers)
+setting. This should be set to a local address on any free port:
+
+```
+db_log_syslog_server: 0.0.0.0:32514
```

Comment:
Since the screenshots below use port `50014`, should we use that here as well for consistency?